nix/deps/gradle/gradle_parser.awk

```diff
@@ -26,7 +26,13 @@ function findPackage(line, regex) {
     # Lines after configuration name list packages
     for (getline line; line != ""; getline line) {
         # React Native is provided by node_modules
-        if (line ~ "com.facebook.react:(react-native|hermes-engine)") { continue }
+        # if (line ~ "com.facebook.react:(react-native|hermes-engine)") { continue }
+        # Example: +--- com.facebook.react:react-native:+ -> com.facebook.react:react-android:0.72.3
+        if (findPackage(line, "--- [^ ]+ -> ([^ :]+):([^ :]+):([^ :]+)$")) {
```

Comment:
Eeeeee... Isn't this already handled by the last `findPackage` call?
https://github.com/status-im/status-mobile/blob/1203834b8c0dacf8931bb71d08a4777ec9a67415/nix/deps/gradle/gradle_parser.awk#L47-L48
What's different about this?

Reply:
I am re-verifying this, will update here.